zebrad/src/components/sync.rs

```diff
@@ -363,9 +363,9 @@ where
                 started_once = true;
             }
 
-            tracing::info!("starting sync, obtaining new tips");
+            info!("starting sync, obtaining new tips");
```

Comment:
I'd like to add a state tip height to this log so we can debug sync issues. I'll open a separate PR.

Reply:
Done in https://github.com/oxarbitrage/zebra/pull/169